Exiled.Loader/Loader.cs

```diff
@@ -404,6 +408,7 @@ private static void LoadDependencies()
             catch (Exception exception)
             {
                 Log.Error($"An error has occurred while loading dependencies! {exception}");
+                ExceptionHandler.TrySendingHelpMessage(exception);
```

Comment:
This needs to be removed. This method is called before Exiled.API.dll is loaded, this this method will not be accessible here, and will cause a MissingMethodException when trying to call the method, which will break dependency and plugin loading completely.